pkg/cmd/cmd.go

```diff
@@ -75,6 +75,12 @@ func RunContext(ctx context.Context, name string, args ...string) (string, error
 	}
 
 	if err = reaper.WaitWrapper(usingReaper, notifyCh, cmd); err != nil {
+		if e, ok := err.(*reaper.ExitError); ok { //nolint:errorlint
```

Comment:
I think one of the linters should warn about it, but it's advised to do `errors.As()` instead of typecasting for errors.